config.json

```diff
@@ -86,6 +86,19 @@
           "quotations",
           "word_definition"
         ]
+      },
+      {
+        "slug": "accumulate",
+        "name": "Accumulate",
+        "uuid": "8a373305-98c9-4b19-9011-085abe19cfed",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 5,
+        "topics": [
+          "quotations",
+          "collections",
+          "combinators"
```

Comment:
Topics were used in existing, v2 tracks. This field has been superseded by `practices` and `prerequisites`. See https://github.com/exercism/docs/blob/main/building/tracks/config-json.md#practice-exercises

Reply:
Interesting - I'll update. Is the general idea in v3 that progression is less linear?